README.rst

```diff
@@ -53,4 +53,4 @@ More documentation
 ------------------
 
 If you would like to download and run the source code of PyTRiPGUI,
-please see `developer documentation <technical.html>`__.
+please see :ref:`developer documentation <technical>`.
```

Comment:
it doesn't work in readme
check here
https://github.com/pytrip/pytripgui/tree/fix/developer-documentation-ref-in-readme

Reply:
You can move main README from RST to MD, as here: https://github.com/APTG/dedx_web

Github itself renders MD much better than the RST format.
Rest of documentation (in docs) can be in RST.

To link between main README and files in the docs folder you can use URL-like links,

Reply:
I think that setting in README.rst this link https://pytrip.github.io/pytripgui/technical.html that takes user to site with documentation is better than these methods:

1. using README.md and README.rst, because then we don't follow DRY
2. using only README.md, because basic Sphinx supports only RST and we will need to install new Sphinx extension (MyST-Parser) through pip

Reply:
I'd simply suggest to:
 1. rename README.rst to README.md
 2. remove docs/README.rst

Main README.md should contain very minimal documentation, 3x times shorter than what we have now (installation section compressed to couple of sentences).
Documentation stored in `docs` should be extensive. It is not necessary to include there (in `docs`) contents of main README.md